src/components/views/Chart.js

```diff
@@ -37,6 +37,9 @@ export default props => {
                     maintainAspectRatio: false,
                     responsive: props.asOverlay,
                     showLines: true,
+                    legend: {
+                        display: false
+                    },
```

Comment:
Maybe we can add a quick comment here explaining that we implement the legend ourselves.